{{cookiecutter.repository_name}}/guild.yml

```diff
@@ -40,6 +40,15 @@
         - model
         - prepare
         - splits
+    contested:
+      main: operations.contested
+      flags-import: all
+      output-scalars: off
+      requires:
```

Comment:
I see that we have a lot of dependencies but otherwise this doesn't really need to be a guild operation right? (I.e. it doesn't have to be reproducible and creating artifacts when we run this op is actually just a downside?)

Does guild have an interface that allows to fetch the model and prepared data instead?

_This would also allow us to skip the contested_display_ hack._
@FelixAbrahamsson Thoughts?